cmd/server/api/handler.go

```diff
@@ -20,12 +20,15 @@ func (h *HTTPClientWrapper) Do(req *http.Request) (*http.Response, error) {
 }
 
 type HandlerInterface interface {
-	PostRPCHandler(w http.ResponseWriter, r *http.Request)
+	PostRPCProxyHandler(w http.ResponseWriter, r *http.Request)
+	UniversalMintingRPCHandler(w http.ResponseWriter, r *http.Request)
+	SetJsonRPCRequest(req JSONRPCRequest)
 }
 
 type Handler struct {
```

Comment:
Usually, if I define in the same package interface and struct that implements that interface I make the interface public and struct the same name but private. 

in this case, it would be 

`type Handler interface`

`type hanlder struct` .

Does it make sense to have the same approach here?